### ISSUE TITLE
calling clear() on an aggregation created with llquantize causes dtrace to exit

### ISSUE THREAD

**Issue by stevenwilliamson** [joyent/illumos-joyent#118]
The below DTrace script reproduces the problem.

Calling clear() on a populated aggregation created with llquantize causes dtrace to abort with the error message:

```
trace: processing aborted: No such file or directory
```

Example script to trigger the issue:

```
#!/usr/sbin/dtrace -s

syscall::read*:entry /fds[arg0].fi_fs == "zfs"/
{
    self->start = timestamp;
}

syscall::read*:return / self->start /
{
    @data["latency_aggregation"] = llquantize(timestamp - self->start, 10, 0, 6, 20);
    self->start = 0;
}

tick-1s
{
  printa(@data);
  clear(@data);
}
```

**Comment by pfmooney** [comment]
Open as [OS-5448](https://smartos.org/bugview/OS-5448).

Thanks

**Comment by pfmooney** [comment]
This should be fixed in 05eb12488a7b3639d543f6458140ecaef9147737
